lib/js/components/Banner/Banner.vue

```diff
@@ -0,0 +1,386 @@
+		#{$self}__buttonTextHorizontal {
+			display: none;
+		}
+
+		#{$self}__buttonTextVertical {
+			display: initial;
+		}
+
+		#{$self}__iconContainer {
+			@include iconContainerVerticalStyles;
+		}
+
+		#{$self}__header {
+			@include headerVerticalStyles;
+		}
+
+		#{$self}__expander {
+			@include expanderVerticalStyles;
+		}
+
+		#{$self}__icon {
+			@include iconVerticalStyles;
+		}
+	}
+
+	&.-warning {
+		background-color: $color-warning-background;
+		border-color: $color-warning-border-weak;
+	}
+
+	&.-success {
+		background-color: $color-success-background;
+		border-color: $color-success-border-weak;
+	}
+
+	&.-info {
+		background-color: $color-info-background;
+		border-color: $color-info-border-weak;
+	}
+
+	&.-fail {
+		background-color: $color-fail-background;
+		border-color: $color-fail-border-weak;
+	}
+
+	&.-neutral {
+		background-color: $color-neutral-background;
+		border-color: $color-neutral-border-weak;
+	}
+
+	&.-default {
+		background-color: $color-default-background;
+		border-color: $color-neutral-border-weak;
+	}
+
+	&.-danger {
+		background-color: $color-danger-background;
+		border-color: $color-danger-border-weak;
+	}
+
+	&__content {
+		display: flex;
+	}
+
+	&__header {
+		@include headerVerticalStyles;
+
+		display: flex;
+		justify-content: space-between;
+		margin-right: $space-xxs;
+		width: 100%;
+
+		@media #{breakpoint-m()} {
+			padding: 0 $space-xxs;
+		}
+	}
+
+	&__title {
+		@include headlineXS;
+		@include textBold;
+	}
+
+	&__defaultText {
+		@include textM;
+	}
+
+	&__text {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		flex-grow: 1;
+		padding: $space-xxxxs 0;
+		margin: 0 $space-s;
+
+		:nth-child(n + 2) {
+			margin-top: $space-xxxxs;
+		}
+	}
+
+	&__iconContainer {
+		@include iconContainerVerticalStyles;
+
+		@media #{breakpoint-m()} {
+			padding: $space-xxs 0;
+		}
+	}
+
+	&__icon {
+		@include iconVerticalStyles;
+
+		border-radius: 100px;
+
+		@media #{breakpoint-m()} {
+			padding: $space-xxs;
+		}
+
+		&.-neutral {
+			color: $color-neutral-icon;
+			background-color: $color-neutral-background-medium;
+		}
+
+		&.-default {
+			color: $color-default-icon;
+			background-color: $color-neutral-background-medium;
+		}
+
+		&.-warning {
+			color: $color-warning-icon;
+			background-color: $color-warning-background-medium;
+		}
+
+		&.-success {
+			color: $color-success-icon;
+			background-color: $color-success-background-medium;
+		}
+
+		&.-info {
+			color: $color-info-icon;
+			background-color: $color-info-background-medium;
+		}
+
+		&.-fail {
+			color: $color-fail-icon;
+			background-color: $color-fail-background-medium;
+		}
+
+		&.-danger {
+			color: $color-danger-icon;
+			background-color: $color-danger-background-medium;
+		}
+	}
+
+	&__buttonTextHorizontal {
+		padding: $space-xs 0;
+		flex-shrink: 0;
+		display: none;
+
+		@media #{breakpoint-m()} {
+			display: initial;
+		}
+	}
+
+	&__buttonTextVertical {
+		padding: $space-xxs 0 0;
```

Comment:
It causes ripple to be moved down:

<img width="537" alt="Screenshot 2022-07-07 at 12 57 43" src="https://user-images.githubusercontent.com/7030884/177758040-54ec60fe-401f-43f1-8e7b-22f82cbeb756.png">

Reply:
margin from nth-child broke the margin. moved margin directly to classes that should have it and it works 🤷🏻‍♂️

Reply:
good catch btw